school/person.rb

```diff
@@ -19,6 +20,10 @@ def of_age?
 
   public
 
+  def add_rental(rental)
+    @rental = rental
+  end
```

Comment:
- [ ] The purpose of the `add_rental` method is to allow the user to create a rental given a book and a date, so this method should be more about the instantiation of the rental class.

  ```
  def add_rental(date, book)
    Rental.new(date, book, self)
  end
  ```